common/subdevice-model.cpp

```diff
@@ -558,10 +559,10 @@ namespace rs2
             }
             else
             {
-                ImGui::PushItemWidth(-1);
+                ImGui::PushItemWidth(170); // Set the width for the combo box itself
```

Comment:
why 170?
What did -1 mean in the past?

Reply:
PushItemWidth(-1) means that the item will get the current window->ItemWidthDefault value. ItemWidthDefault is set automatically when opening the window based on the window size.
I changed it back to ImGui::PushItemWidth(-1);